examples/Retrieval-Models.ipynb

```diff
@@ -0,0 +1,1528 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    schema = TensorflowMetadata.from_proto_text_file('/workspace/retrieval/processed/train/').to_merlin_schema()
</pre><p> </p><p>Rather than load up the schema ourselves, can we just load up the Dataset - and use the schema from that?</p><p> </p><pre class="ql-syntax" spellcheck="false">train_dataset = Dataset("/workspace/retrieval/processed/train/", engine="parquet")
schema = train_dataset.schema
</pre><p> </p><p>We can then use the train_dataset for fitting the model as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>